lib/src/models/issue.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:meta/meta.dart';
+import 'package:source_span/source_span.dart';
+
+import 'replacement.dart';
+import 'severity.dart';
+
+/// Represents a issue detected by the analysis rule
+@immutable
+class Issue {
+  /// The id of the rule associated with this issue
+  final String ruleId;
+
+  /// The url of a page containing documentation associated with this issue
+  final Uri documentation;
+
+  /// The source location associated with this issue
+  final SourceSpan location;
+
+  /// The severity of this issue
+  final Severity severity;
+
+  /// Short message (single line)
+  final String message;
+
+  /// Verbose message containing information about how the user can fix this issue
+  final String verboseMessage;
```

Comment:
Correction: `The verbose message containing information about how the user can fix this issue`